low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -463,9 +465,61 @@ def process(self):
 
 
 class CleanupCmd(Cmd):
-    """Removes SSPL configs."""
+
+    """Restores the default SSPL configs."""
 
     name = "cleanup"
+    product = None
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def validate(self):
+        # Validate config inputs
+        from framework.utils.utility import Utility
+        Conf.load(GLOBAL_CONFIG_INDEX, global_config_path)
+        self.product = Utility.get_config_value(GLOBAL_CONFIG_INDEX,
+            "cortx>release>product")
+        if self.product is None:
+            msg = "%s - validation failure. %s" % (
+                self.name, "'Product' name is required.")
+            logger.error(msg)
+            raise SetupError(errno.EINVAL, msg)
+        logger.info("%s - Validation done" % self.name)
+
+    def process(self):
+        try:
+            if os.path.exists(file_store_config_path):
+                os.remove(file_store_config_path)
+            shutil.copyfile("%s/conf/sspl.conf.%s.yaml" % (SSPL_BASE_DIR,
+                self.product), file_store_config_path)
+            logger.info("%s - Process done" % self.name)
+        except OSError as e:
+            logger.error(f"Failed in Cleanup. ERROR: {e}")
+
+
+class BackupCmd(Cmd):
+
+    """Backup support for SSPL componenet."""
+
+    name = "backup"
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def validate(self):
+        # Common validator classes to check Cortx/system wide validator
+        pass
+
+    def process(self):
+        pass
```

Comment:
Add a log stating not implemented. same in restore.process()

Reply:
Done